shared/fs/browser/rows/load-files-when-needed.tsx

```diff
@@ -89,9 +87,6 @@ class LoadFilesWhenNeeded extends React.PureComponent<Props> {
   }
 }
 
-export default namedConnect(
-  mapStateToProps,
-  mapDispatchToProps,
-  mergeProps,
-  'LoadFilesWhenNeeded'
-)(LoadFilesWhenNeeded) as any
+export default namedConnect(mapStateToProps, mapDispatchToProps, mergeProps, 'LoadFilesWhenNeeded')(
+  LoadFilesWhenNeeded
+) as any
```

Comment:
if you're in here you might want to fix this

Reply:
oh hmm this must be my prettier. I didn't actually touch it. Will try to fix it anyway

Reply:
oh wait I did touch this file. 🤦🏻‍♂️

Reply:
I'm gonna rebase https://github.com/keybase/client/pull/17684 over this, and fix a Files bug there. That gets rid of this component, so I'll just leave this for now.